.eslintrc.js

```diff
@@ -62,6 +62,7 @@ module.exports = defineConfig({
   },
 
   rules: {
+    'object-curly-spacing': ['error', 'always'],
```

Comment:
@renchap so i guess this can be helpful

Reply:
You should not change this in this PR, as it creates issue in other files that are badly formatted. Can you please remove this file from the PR, and we will merge it?
Thanks!

Reply:
sure, done